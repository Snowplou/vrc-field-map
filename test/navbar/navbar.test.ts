import { cleanup, render } from "@testing-library/svelte";
import { describe, it, expect, afterEach } from "vitest";
import Navbar from "/src/lib/navbar/NavBar.svelte";

describe("Navbar.svelte", () => {
    // TODO: @testing-library/svelte claims to add this automatically but it doesn't work without explicit afterEach
    afterEach(() => cleanup());

    it("mounts", () => {
        const { container } = render(Navbar);
        expect(container).toBeTruthy();
        expect(container.innerHTML).toMatchSnapshot();
    });
});