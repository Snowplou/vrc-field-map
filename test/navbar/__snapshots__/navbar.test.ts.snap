// Vitest Snapshot v1

exports[`Navbar.svelte > mounts 1`] = `"<div><div class=\\"navbar-wrapper svelte-1i59wqc\\"><div class=\\"navbar-main svelte-1i59wqc\\"><span class=\\"svelte-pbnhkd\\">Spin Up</span> <span class=\\"svelte-pbnhkd\\">Tipping Point</span></div></div></div>"`;
